### ISSUE TITLE
Conflicting documentation about default value of pillar_opts

### ISSUE THREAD

**Issue by Arabus** [saltstack/salt#38629]
### Description of Issue/Question
The Saltstack documentation about pillar data including the master configuration by default differs between the walkthrough and the global documentation.

In https://docs.saltstack.com/en/latest/topics/pillar/index.html
```
Master Config in Pillar

For convenience the data stored in the master configuration file can be made available in all minion's pillars. This makes global configuration of services and systems very easy but may not be desired if sensitive data is stored in the master configuration. This option is disabled by default.

To enable the master config from being added to the pillar set pillar_opts to True in the minion config file:

pillar_opts: True
```

In https://docs.saltstack.com/en/latest/topics/tutorials/pillar.html
```
By default the contents of the master configuration file are loaded into pillar for all minions. This enables the master configuration file to be used for global configuration of minions.
```

This is from the latest documentation, but I guess it persists throughout the older versions. in Boron at least the default value is indeed False.

**Comment by gtmanfred** [comment]
The pillar_opts is set to False by default.

If you wouldn't mind opening a PR to fix this documentation, we would greatly appreciate it!, otherwise I have tagged it as a documentation issue to get fixed.

Thanks,
Daniel

**Comment by rallytime** [comment]
Good catch @Arabus. I have submitted a documentation fix for this issue in PR #38792.
